lib/fastlane/plugin/test_center/actions/collate_test_result_bundles.rb

```diff
@@ -19,6 +22,56 @@ def self.run(params)
         UI.message("Finished collating test_result bundle to '#{params[:collated_bundle]}'")
       end
 
+      def self.xcresulttool_merge(params)
+        test_result_bundlepaths = params[:bundles]
+        found_version3_format_bundle = test_result_bundlepaths.any? do |test_result_bundlepath|
+          is_bundle_format_3?(test_result_bundlepath)
+        end
+
+        
+        if found_version3_format_bundle
+          `xcrun xcresulttool version 2> /dev/null`
+          unless $?.exitstatus.zero?
+            UI.user_error!("""
+              Unable to collate version 3 format test_result bundle without the xcrun xcresulttool.
+              Please install and select Xcode 11, and then run the command again.""")
+          end
+          collated_bundlepath = File.expand_path(params[:collated_bundle])
+          Dir.mktmpdir do |dir|
+            tmp_xcresult_bundlepaths = []
+            test_result_bundlepaths.each do |test_result_bundlepath|
+              bundlename = File.basename(test_result_bundlepath)
+              tmp_xcresult_bundlepath = Dir.mktmpdir([bundlename, '.xcresult'])
+              FileUtils.rmdir([tmp_xcresult_bundlepath])
+              FileUtils.symlink(test_result_bundlepath, "#{tmp_xcresult_bundlepath}", force: true)
```

Comment:
add comment explaining that we have to appease the `xcrun xcresulttool's` requirement that the bundle ends with the `xcresult` extension.